veocean-data-farming/README.md

```diff
@@ -0,0 +1,34 @@
+---
+description: An overview of Ocean Protocol's governance and incentives mechanisms
+---
+# veOCEAN & Data Farming
+
+veOCEAN is a fork of veCRV, and enables you to become a governance participant, eligible to receive rewards and engage with different protocol mechanisms.
+
+The following docs should provide you with sufficient intuition to access, utilize, and build upon the protocol's core incentive and reward system: Data Farming.
+
+![](./images/df_rewards_page.png)
+
+## veOCEAN
+
+Learning about [veOCEAN](veocean.md) will help you answer the question `"What can I do with my veOCEAN?"` and give you insights on how veOCEAN works. It will teach you everything you need to know about why it exists and how it works.
```

Comment:
```suggestion
Learning about [veOCEAN](veocean.md) will help you answer the question "What can I do with my veOCEAN?" and give you insights on how veOCEAN works. It will teach you everything you need to know about why it exists and how it works.
```

Reply:
Done